src/pact.spec.ts

```diff
@@ -237,27 +235,26 @@ describe("Pact Spec", () => {
 	});
 
 	// These tests never worked because the expect was wrong.  When fixed, massive issue ensues
-	describe.skip("Verify Pacts", () => {
-		context("With provider states", () => {
-			it("should start the pact-provider-verifier service and verify pacts", () => {
-				let opts = {
-					providerBaseUrl: "http://localhost",
-					pactUrls: [path.dirname(currentDir)]
-				};
-				return expect(pact.verifyPacts(opts)).to.eventually.be.fulfilled;
-			});
-		});
-	});
-
-	describe.skip("Publish Pacts", () => {
-		it("should start running the Pact publishing process", () => {
-			let opts = {
-				pactBroker: "http://localhost",
-				pactFilesOrDirs: [path.dirname(currentDir)],
-				consumerVersion: "1.0.0"
-			};
-			return expect(pact.publishPacts(opts)).to.eventually.be.fulfilled;
-		});
-	});
-})
-;
+	/*describe.skip("Verify Pacts", () => {
```

Comment:
Let's just remove these - they are very much integration tests that belong in the other project anyhow.

Reply:
We also have tests in the integration folder for them.

Reply:
ah cool, will do.